### ISSUE TITLE
Wrong name when file does already exist 

### ISSUE THREAD

**Issue by Shusek** [anggrayudi/SimpleStorage#92]
Library version: 1.4.0
OS version: API 30
Device model: Emulator

**Describe the bug**
When creating a new file while a file with the same name does already exist, the "(2)" is added at the wrong place:

 'Some_File.pdf (2) instead 'Some_File (2).pdf'.

**To Reproduce**
Bug cannot be reproduced on API 24.

File is created by:
```
  DocumentFileCompat.createDownloadWithMediaStoreFallback(
            context,
            FileDescription(fileName)
        )?.uri
            ?.toDocumentFile(context)
```

Other:
This regression appeared after the update from 0.14.0 to 1.4.0 version.

**Comment by anggrayudi** [comment]
Did not happen to me. Everything works fine:

<img width="1728" alt="Screen Shot 2022-07-15 at 13 35 23" src="https://user-images.githubusercontent.com/12292380/179165813-c9e80776-2a78-4d13-9b02-3b4f6f4ea939.png">

Even creating files with `makeFile()` have no problem:

<img width="1728" alt="Screen Shot 2022-07-15 at 13 33 37" src="https://user-images.githubusercontent.com/12292380/179165870-31d25ffb-4e96-468d-9739-ea7b7f6f4620.png">

<img width="1728" alt="Screen Shot 2022-07-15 at 13 34 09" src="https://user-images.githubusercontent.com/12292380/179165987-d01ce83b-2cff-447c-8230-2bfb078e03a5.png">

**Comment by anggrayudi** [comment]
For now, the bug can be avoided by setting the mime type explicitly:

```kotlin
val desc = FileDescription("Some_File.pdf", "", "application/pdf")
val filePdf = DocumentFileCompat.createDownloadWithMediaStoreFallback(this, desc)?.uri?.toDocumentFile(this)
```

Future improvements, maybe v1.4.1 will add automatic mime type detection, so you don't need to add mime type into the argument anymore.

**Comment by anggrayudi** [comment]
Fixed in [v1.4.1](https://github.com/anggrayudi/SimpleStorage/releases/tag/1.4.1)
